src/constants/oracleConfig.ts

```diff
@@ -20,81 +10,252 @@ const optimisticV2Chains: Record<
   oracle.types.state.PartialChainConfig
 > = {};
 
-// enable local node if debug is on
+// enable local node if debug is on, this only works as a fork of mainnet
 if (process.env.REACT_APP_DEBUG) {
-  optimisticChains[mmChainId] = {
+  const chainId = ChainId.MM_TESTNET;
+  const chain = CHAINS[chainId];
+  const config: oracle.types.state.PartialChainConfig = {
     rpcUrls: ["http://127.0.0.1:8545"],
-    nativeCurrency: CHAINS[ethChainId].nativeCurrency,
-    blockExplorerUrls: [CHAINS[ethChainId].explorerUrl],
-    chainName: CHAINS[ethChainId].name,
+    nativeCurrency: chain.nativeCurrency,
+    blockExplorerUrls: [chain.explorerUrl],
+    chainName: chain.name,
+  };
+  optimisticChains[chainId] = {
+    ...config,
     multicall2Address: "0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696",
     optimisticOracleAddress: "0xC43767F4592DF265B4a9F1a398B97fF24F38C6A6",
   };
+  skinnyChains[chainId] = {
+    ...config,
+    multicall2Address: "0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696",
```

Comment:
Why is this address specified in debug mode but not elsewhere?

Reply:
debug mode should be a fork of another chain, but internally we use chain id to identify known addresses, but we do not have it for the debug chain id (31337, or something likethat), since technically you could fork any chain, so you have to specify the addresses manually.

Reply:
i changed the environement variable to be separate from DEBUG ( which enables logs).